vita3k/kernel/src/thread/thread.cpp

```diff
@@ -116,11 +116,11 @@ SceUID create_thread(Ptr<const void> entry_point, KernelState &kernel, MemState
 
     alloc_name = fmt::format("TLS for thread {} (#{})", name, thid);
 
-    auto tls_address = alloc(mem, 0x800, alloc_name.c_str()) + 0x800;
+    Ptr<void> tls_address_ptr = Ptr<void>(alloc(mem, 0x800, alloc_name.c_str()));
```

Comment:
you can also using auto here, you using Ptr for intial value so is good on auto too :)